packages/babel-helpers/src/helpers.js

```diff
@@ -943,7 +943,7 @@ helpers.iterableToArrayLimit = helper("7.0.0-beta.0")`
     // _e = _iteratorError
     // _i = _iterator
     // _s = _step
-
+    if (!Symbol.iterator in Object(arr)) return;
```

Comment:
The `iterableToArray` has guarded this check with an `arguments` check
```js
export default function _iterableToArray(iter) {
    if (
      Symbol.iterator in Object(iter) ||
      Object.prototype.toString.call(iter) === "[object Arguments]"
    ) return Array.from(iter);
  }
```

But 
```js
(function () { return Symbol.iterator in Object(arguments) })()
```
returns true in Chrome 76. I am not sure if this behavior is inconsistent across different browsers.

Reply:
Oh, there is a browser support matrix on `arguments[@@iterator]`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments/@@iterator